.npmrc

```diff
@@ -6,4 +6,6 @@ hoist = false
 
 # We _do_ try to hoist `@types` packages so that TS will not end up with type
 # errors from comparing types from different versions of the same package.
-public-hoist-pattern=['@types/*']
\ No newline at end of file
+public-hoist-pattern=['@types/*']
+
+auto-install-peers=true
```

Comment:
What would it look like to install peer deps explicitly? Is it possible to see which peer deps are currently being auto installed? I have a slight leaning towards more explicitness when it comes to package configuration, since it can already be confusing.

Reply:
Yeah, this nudge is useful - very much leaning toward explicit imports as well, so I'm going to test that out locally and see if we can avoid this config change

Reply:
Spent way too long trying to untangle the peer dependency mess, but the repo is in a bit of a brittle state and it was a can of worms trying to get it working. After digging a little more into the breaking changes pnpm introduced between v6 and v7, it looks like the previous behaviour was similar to setting `auto-install-peers` to true, so I'm going to leave it in place for now and see if we can get stricter in the future.

Reply:
Ok, thanks for digging into that!